streampipes-client-python/streampipes/functions/utils/data_stream_generator.py

```diff
@@ -83,6 +94,19 @@ def create_data_stream(name: str, attributes: Dict[str, str], stream_id: Optiona
         ]
     )
 
-    if not stream_id:
-        return DataStream(name=name, event_schema=event_schema)
-    return DataStream(element_id=stream_id, name=name, event_schema=event_schema)
+    transport_protocols = [TransportProtocol()]
+    if broker == SupportedBroker.KAFKA:
+        transport_protocols = [
+            TransportProtocol(
+                class_name="org.apache.streampipes.model.grounding.KafkaTransportProtocol",  # type: ignore
```

Comment:
muting mypy shouldn't be necessary here

Reply:
Because `class_name` is an alias from pydantic, mypy says that it doesn't exists

Reply:
Okay, we leave it for now, but it should work :)